modules/cnf-configuring-llc-aware-cpu-pinning.adoc

```diff
@@ -0,0 +1,282 @@
+// Module included in the following assemblies:
+//
+// * scalability_and_performance/cnf-tuning-low-latency-nodes-with-perf-profile.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="cnf-configuring-llc-aware-cpu-pinning_{context}"]
+= Configuring LLC-aware CPU pinning for low latency workloads
+
+Traditional CPU architectures feature a single last-level cache (LLC), or layer 3 cache, which all cores on the CPU can use. Some modern CPU architectures split this LLC into two or more physical caches to reduce the distance, and therefore latency, between the cores and the LLC.
+
+To take advantage of this split LLC architecture, you can configure a CPU Manager policy to pin workloads to cores that have the lowest latency to the LLC. This configuration can significantly improve performance for low latency workloads. In split LLC architecture CPUs, this approach also reduces noisy neighbor conditions and inter-cache contention issues that might otherwise occur without LLC-aware scheduling.
+
+.LLC-aware CPU pinning
+image::../images/llc-aware-pinning.png[LLC-aware pinning]
```

Comment:
please add a note about the fact this is an illustrative example and does not necessarily reflect the actual behavior of the cpumanager allocator